lib/matplotlib/tests/test_axes.py

```diff
@@ -3078,6 +3078,18 @@ def layers(n, m):
 
     fig, axs = plt.subplots(2, 2)
 
+    # Attempt to use an integer value for 'baseline'
+    try:
+        axs[0, 0].stackplot(range(100), d.T, baseline=5)
+    except ValueError as e:
+        print("Error when using an integer as baseline:", e)
```

Comment:
If you are testing something that should fail the test if the error is raised, then remove the try/except.

If you are testing that this _should_ raise, then use `with pytest.raises(...)` which will ensure that the exception is actually raised (where printing will just ignore whether or not an exception was raised)